### ISSUE TITLE
editOnClick() not working

### ISSUE THREAD

**Issue by Bradzer** [Power-Components/livewire-powergrid#174]
**Describe the bug**

Editing a column configured with `editOnClick()` then hitting `<Enter>` does NOT update the cell.
It seems like the function `sendEventInputChanged()` cannot be accessed.
Here is what the dev tools console reports:
```
Alpine Expression Error: sendEventInputChanged is not defined

Expression: "sendEventInputChanged($event, id, field); editable = false; content = htmlSpecialChars($event.target.value)"
```

**To Reproduce**
N/A

**Laravel version (exactly):**
* 8.69.0
 
**Alpine version:**
* 3.7.0

**Livewire version:**
* 2.7.2

**Theme:**

- [x] Bootstrap
   - Version:  Default Bootstrap 5 used by Powergrid

**Comment by luanfreitasdev** [comment]
Hi @Bradzer . This is weird. I was unable to reproduce this error. 

Could you check if this function exists in the html?

https://user-images.githubusercontent.com/33601626/145117717-6899d889-d3c6-4b93-943d-d2459300970e.mp4

**Comment by Bradzer** [comment]
Thank you @luanfreitasdev for your reply.

I think I found what the problem was.

In fact, I do not generate the table on first load. I generate the table on subsequent requests. And because scripts only get loaded on first load, I don't have access to the necessary script `sendEventInputChanged()` even though it's added (but of course not processed as it's too late) in the page.

The workaround I was using when I started the issue was to add the function `sendEventInputChanged` in view so that the function gets loaded on first render.

Is there a way I could deal with that ?
